doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -271,10 +280,24 @@ nRF Desktop
     * Set the max compiled-in log level to ``warning`` for the Non-Volatile Storage (:kconfig:option:`CONFIG_NVS_LOG_LEVEL`).
     * Lowered a log level to ``debug`` for the ``Identity x created`` log in the :ref:`nrf_desktop_ble_bond`.
 
-  * By default, nRF Desktop dongles use the Bluetooth appearance (:kconfig:option:`CONFIG_BT_DEVICE_APPEARANCE)` of keyboard.
-    The new default configuration value improves consistency with the used HID boot interface.
   * The default values of the :kconfig:option:`CONFIG_BT_GATT_CHRC_POOL_SIZE` and :kconfig:option:`CONFIG_BT_GATT_UUID16_POOL_SIZE` Kconfig options are tailored to the nRF Desktop application requirements.
   * The :ref:`nrf_desktop_fast_pair_app` to remove the Fast Pair advertising payload for the dongle peer.
+  * Default values of Bluetooth device name (:kconfig:option:`CONFIG_BT_DEVICE_NAME`) and Bluetooth device appearance (:kconfig:option:`CONFIG_BT_DEVICE_APPEARANCE`) are set relying respectively on nRF Desktop product name or nRF Desktop device role and type combination.
+  * By default, nRF Desktop dongles use the Bluetooth appearance (:kconfig:option:`CONFIG_BT_DEVICE_APPEARANCE)` of keyboard.
+    The new default configuration value improves consistency with the used HID boot interface.
+  * Disabled USB remote wakeup (:kconfig:option:`CONFIG_USB_DEVICE_REMOTE_WAKEUP`) in MCUboot bootloader configurations.
+    The functionality is not used by the bootloader.
+  * :ref:`nrf_desktop_hids` registers the GATT HID Service before Bluetooth LE is enabled.
+    This is done to avoid submitting works related to Service Changed indication and GATT database hash calculation before system settings are loaded from non-volatile memory.
```

Comment:
```suggestion
    This is done to avoid submitting works related to Service Changed indication and GATT database hash calculation before the system settings are loaded from non-volatile memory.
```